nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -433,6 +433,11 @@
   be converted to UTF-8 unless the `useUtf8` package option is enabled; UTF-8
   converted dictionaries will have the .utf8 suffix appended to its filename.
 
+- The `earlyoom` service is now using upstream systemd service, which enables
```

Comment:
24.11 is out...

Reply:
Rebased and moved it to 25.05 release note.